Chapter19/02_play.py

```diff
@@ -23,7 +24,7 @@
 
     env = gym.make(args.env)
     if args.record:
-        env = gym.wrappers.Monitor(env, args.record)
+        env = wrappers.Monitor(env, args.record)
```

Comment:
Not sure about this change. What's the point?

Reply:
https://github.com/dennybritz/reinforcement-learning/issues/63

I get this same issue (under Python3.6.9, Ubuntu 18.04), with successful fix using CurrenJ's solution.  I';m not sure why.